python/pyspark/pandas/frame.py

```diff
@@ -3582,14 +3571,18 @@ def between_time(
         if not isinstance(self.index, ps.DatetimeIndex):
             raise TypeError("Index must be DatetimeIndex")
 
+        if inclusive not in ["left", "right", "both", "neither"]:
+            msg = "Inclusive has to be either 'both', 'neither', 'left' or 'right'"
+            raise ValueError(msg)
```

Comment:
Can we use `PySparkValueError` and error classes?

Reply:
Changed to PySparkValueError and the new error_class has been added